node/core/dispute-coordinator/src/metrics.rs

```diff
@@ -19,12 +19,16 @@ use polkadot_node_subsystem_util::metrics::{self, prometheus};
 #[derive(Clone)]
 struct MetricsInner {
 	/// Number of opened disputes.
+	#[cfg(feature = "disputes")]
```

Comment:
More stuff to remove when we get rid of the feature gate :P

Reply:
Yeah, ...